addons/forum-time-zones/userscript.js

```diff
@@ -0,0 +1,28 @@
+import moment from "../../libraries/moment.js";
+export default async function ({ addon, _global, _console }) {
```

Comment:
Just curious, why the underscores in `_global` and `_console`?

Reply:
I thought it would break moment.js, due too Node.js and stuff.

Reply:
Nah that won't cause problems here, because `global` isn't in the global (ironic)